src/pq/param.cr

```diff
@@ -68,6 +68,11 @@ module PQ
       text string
     end
 
+    def self.encode(val : PG::Interval)
+      # https://www.postgresql.org/docs/current/datatype-datetime.html#DATATYPE-INTERVAL-INPUT
+      text "#{val.months} months #{val.days} days #{val.microseconds} microseconds"
```

Comment:
Alternatively, could move this logic into `PG::Interval#to_s(io)`?

Reply:
How come no `minutes` or `seconds`? If I had something like this

```crystal
db.exec("INSERT INTO events (recurring_interval) VALUES (?)", args: 5.minutes)
```

Wouldn't this insert `0 months 0 days 0 microseconds`? Or is that unrelated to what this PR is for?

Reply:
I wondered about this too. But as far as I can tell the `PG::Interval` type only exposes the three properties I used. However your example would work just fine as `Time::Span` also works just fine for inserting.

I could see it making sense to also handle the opposite case of #272 where we allow reading `INTERVAL` PG types into a `Time::Span`. Then for most use cases you could use that, but if you really need the exact same behavior/precision you could use `PG::Interval`.